### ISSUE TITLE
Light background theme for the CLI

### ISSUE THREAD

**Issue by Poroing** [Megabyte918/MultiOgar-Edited#809]
Is it possible to have some way to change the CLI, a command line argument for example, so that it is readable on light background?

If that is alright I might as well make this my first Open Source contribution if that is something that is alright for a starter to do ^^

**Comment by MastaCoder** [comment]
Sure, feel free to do so :)

**Comment by Poroing** [comment]
I changed the `Logger.js` and `index.js` so that the CLI coloscheme is changed for a light background on the presence of the flags `-l` or `--light-backgound`, I will create a pull request after my questions are cleared ^^

I tried to keep the coding style of the project int the code I wrote, is it something that I should always do? Or is it alright to come in and wrote things in my own style for a project that doesn't give any instruction for contribution?

Maybe I should not care this much about License but, I saw that the `Logger.js` is under the Apache License so I went to read it, and it is stipulated that I should write down that I added some modification to the file but it seems that there already have been some contributors who modified the file, according to GitHub, without writing that they modified the file in the file, does the information GitHub give is sufficient or should I wrote my pseudo somewhere in the file?

Since I'm beginning in Open Source, I have a lot of question ^^ And I'm starting to notice that I greatly lack in English vocabulary!
